plugins/Quotes/class.quotes.plugin.php

```diff
@@ -544,6 +541,20 @@ protected function formatQuote($type, $id, &$quoteData, $format = false) {
         Emoji::instance()->enabled = $emojiEnabled;
     }
 
+    /**
+     * Checks if the user can view discussion.
+     *
+     * @param object $discussion
+     * @throws Exception If the user cannot view the discussion.
```

Comment:
I know this is what `permissionException` documents as its return type. While technically true, you are actually throwing an instance of `Gdn_UserException`, specifically. We should document specific exceptions, whenever possible. It's probably worth updating the doc block of `permissionException` to also reflect the proper return value.